src/engine/GameState.java

```diff
@@ -205,4 +206,12 @@ public final void setLivesRecovery() {
 	public final int getBulletsRemaining() {
 		return BulletsRemaining;
 	}
+	public Color getPlayerColor(){
+		Random r = new Random();
+		Color color = new Color(r.nextInt(255),r.nextInt(255),r.nextInt(255));
+		if(!(color.equals(Color.YELLOW) || color.equals(Color.BLUE) || color.equals(Color.RED) || color.equals(Color.CYAN)))
```

Comment:
[P2]

If the purpose is to generate a random number in the range from 0 to 255, 
it is correct to write ```r.nextInt(256)```.

It may be difficult for users to feel the difference in the actual game, 
but I would appreciate it if you could review it.